docs/_static/network/v1.yaml

```diff
@@ -9,6 +9,23 @@ servers:
   - url: http://localhost:1323
 paths:
   /internal/network/v1/transaction:
+    parameters:
```

Comment:
The TODO in the description can go now.

       > Lists all transactions on the DAG. Since this call returns all transactions on the DAG, care should be taken when there
        are many of them.
        TODO: By then we'd need a more elaborate querying interface (ranging over timestamps/hashes, pagination, filtering, etc).


Also adjust the "all transactions on the DAG" description